CONTRIBUTING.md

```diff
@@ -45,9 +45,86 @@ include your changes.
   the name of the thing you're changing in at the beginning of the message,
   followed by a colon: the plugin name for plugins, "docs" for documentation
   files, "coretasks" for `coretasks.py`, "db" for the database feature, etc.
-* Python files should always have  `from __future__ import generator_stop`
+* Python files should always have `from __future__ import annotations`
   as the first line after the module docstring.
 
+Running from source
+-------------------
+
+Sopel is your typical Python project: you can install it from source, edit
+the code using standard tooling, and take advantage of your favourite code
```

Comment:
I _think_ American spellings are pretty consistent throughout the docs. Not necessarily a preference, but I don't think anyone wants to go through all the documentation and change _everything_ to the UK spellings.

Reply:
Oh, it's "favorite", right? You'll have to point me out these things, because I'm not sure I'm even aware of all of them.

Reply:
```suggestion
the code using standard tooling, and take advantage of your favorite code
```